frontend/src/component/admin/groups/GroupForm/GroupFormUsersSelect/GroupFormUsersSelect.tsx

```diff
@@ -99,48 +215,112 @@ export const GroupFormUsersSelect: VFC<IGroupFormUsersSelectProps> = ({
             }),
     ];
 
-    return (
-        <StyledGroupFormUsersSelect>
-            <Autocomplete
-                data-testid={UG_USERS_ID}
-                size="small"
-                multiple
-                limitTags={1}
-                openOnFocus
-                disableCloseOnSelect
-                value={users as UserOption[]}
-                onChange={(event, newValue, reason) => {
-                    if (
-                        event.type === 'keydown' &&
-                        (event as React.KeyboardEvent).key === 'Backspace' &&
-                        reason === 'removeOption'
-                    ) {
-                        return;
+    const virtualized = true;
+    if (virtualized)
```

Comment:
We can have a threshold to decide when to show the virtualized dropdown list (e.g. if length > 200 show virtualized), but this requires us to maintain two implementations. Also, we can have a feature flag to always display this.

I think we need to abstract this in a reusable component, but as a first step this could be good enough to prove the solution.